### PR TITLE
Minor tweaks

### PR DESCRIPTION
**Pull request by shvaikalesh**
Probably not the most useful PR, but still, worth a shot IMO.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,16 +3,12 @@
...
-    "url": "https://github.com/DmitrySoshnikov/regexp-tree.git"
-  },
+  "repository": "DmitrySoshnikov/regexp-tree",
   "homepage": "https://github.com/DmitrySoshnikov/regexp-tree",
   "bugs": "https://github.com/DmitrySoshnikov/regexp-tree/issues",
-  "main": "index.js",
   "scripts": {
     "build": "./scripts/build",
-    "test": "./scripts/test",
+    "test": "jest",
```
Comment (DmitrySoshnikov) [inline package.json]: Will this work if no global `jest` is installed? The purpose of the script was to use the local version of `jest` installed from `devDependencies`.

Comment (shvaikalesh) [inline package.json]: Yep, `npm` prepends ([docs](https://docs.npmjs.com/misc/scripts#path)) `node_modules/.bin` to `PATH` before running the code, so it does not require `jest` installed globally. Travis passed, and it does not have global `jest`.

Comment (DmitrySoshnikov) [inline package.json]: I see, didn't know about this. Sounds good!

**Comment by DmitrySoshnikov** [comment]
OK, looks good overall!

**Pull request merged**